src/mendeleyMain.vb

```diff
@@ -692,6 +692,35 @@ EndOfSub:
     uiDisabled = False
 End Sub
 
+'Remove Bookmarks after export to MS-WORD
+Sub Remove_Bookmark()
+On Error GoTo ErrorHandler
```

Comment:
Have a look how the ErrorHandler is used in the plugin - I think that we should use the same approach (if not debug: executes reportError, if debug then no error handler which helps us fixing things instead of ignoring the errors)